rskj-core/src/test/java/co/rsk/pcc/AltBN128Test.java

```diff
@@ -0,0 +1,718 @@
+package co.rsk.pcc;
+
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2017 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+import co.rsk.config.TestSystemProperties;
+import co.rsk.config.VmConfig;
+import co.rsk.helpers.Stopwatch;
+import co.rsk.vm.BytecodeCompiler;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ConsensusRule;
+import org.ethereum.core.BlockFactory;
+import org.ethereum.vm.DataWord;
+import org.ethereum.vm.PrecompiledContracts;
+import org.ethereum.vm.VM;
+import org.ethereum.vm.program.Program;
+import org.ethereum.vm.program.Stack;
+import org.ethereum.vm.program.invoke.ProgramInvokeMockImpl;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.math.BigInteger;
+import java.util.HashSet;
+
+import static org.ethereum.util.ByteUtil.stripLeadingZeroes;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author Sebastian Sicardi
+ * @since 10.09.2019
+ */
+
+public class AltBN128Test {
+    private final TestSystemProperties config = new TestSystemProperties();
```

Comment:
I think we prefer to initialize fields in the constructor

Reply:
It's a testing class, they usually don't have a constructor, do you want me to do one anyway?